docs/conf.py

```diff
@@ -0,0 +1,84 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+sys.path.insert(0, os.path.abspath('..'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Gro API Client'
+html_logo = '_images/logo.jpg'
+copyright = '2019, Gro Intelligence'
+author = 'Gro Intelligence'
+
+# The full version, including alpha/beta/rc tags
+release = '1.19.1'
```

Comment:
is this the same version number as in setup.py?

Reply:
It's the version that was there when I started working on this docs PR, but now we're up to 1.19.3. It's not being used anywhere, so I just left it. Trying removing it now

Reply:
Works fine without it https://app.shippable.com/github/gro-intelligence/api-client/runs/426/1/console